docs/source/conf.py

```diff
@@ -1,26 +1,258 @@
+# non-false value, then it is used:
+# today = ''
+# Else, today_fmt is used as the format for a strftime call.
+# today_fmt = '%B %d, %Y'
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+exclude_patterns = ["build", "_templates/*.rst'", "Thumbs.db", ".DS_Store"]
+
+# The reST default role (used for this markup: `text`) to use for all
+# documents.
+# default_role = None
+
+# If true, '()' will be appended to :func: etc. cross-reference text.
+# add_function_parentheses = True
+
+# If true, the current module name will be prepended to all description
+# unit titles (such as .. function::).
+# add_module_names = True
+
+# If true, sectionauthor and moduleauthor directives will be shown in the
+# output. They are ignored by default.
+# show_authors = False
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = "sphinx"
+
+# A list of ignored prefixes for module index sorting.
+# modindex_common_prefix = []
+
+# If true, keep warnings as "system message" paragraphs in the built documents.
+# keep_warnings = False
+
+# If true, `todo` and `todoList` produce output, else they produce nothing.
+todo_include_todos = True  # set False by default, enable for debugging
+
+
+# -- Options for HTML output ----------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+# html_theme = "sphinx_rtd_theme"
+html_theme = "pydata_sphinx_theme"
+
+# Theme options are theme-specific and customize the look and feel of a theme
+# further.  For a list of options available for each theme, see the
+# documentation.
+#
+html_theme_options = {
+    # "google_analytics_id": "UA-140243896-1",
+    "show_prev_next": False,
+    "github_url": "https://github.com/equinor/iterative_ensemble_smoother",
+    "icon_links": [
+        {
+            "name": "Support",
+            "url": "https://github.com/equinor/iterative_ensemble_smoother/issues",
+            "icon": "fa fa-comment fa-fw",
+        },
+        # {
+        #     "name": "The Paper",
+        #     "url": "https://doi.org/10.21105/joss.01450",
+        #     "icon": "fa fa-file-text fa-fw",
+        # },
+    ],
+}
+
+# Bibliography
+bibtex_bibfiles = ["./refs.bib"]
+bibtex_default_style = "unsrt"
+bibtex_reference_style = "author_year"
+suppress_warnings = ["bibtex.duplicate_citation", "autosectionlabel.*"]
 
 
 numpydoc_class_members_toctree = False
+
+# Issue with attributes section, see:
+# https://github.com/sphinx-doc/sphinx/issues/2115
+# Solution:
+# https://michaelgoerz.net/notes/extending-sphinx-napoleon-docstring-sections.html
+# -- Extensions to the  Napoleon GoogleDocstring class ---------------------
+
+
+def parse_keys_section(self, section):
+    return self._format_fields("Keys", self._consume_fields())
+
+
+GoogleDocstring._parse_keys_section = parse_keys_section
```

Comment:
Remove google stuff, since we use NumPy docstrings.

Reply:
Removed.